README.md

```diff
@@ -36,26 +39,73 @@ pip install tensorflow tensorflow-probability gpflow
 pip install torch gpytorch
 ```
 
+- JAX (the cpu version)
+```
+pip install "jax[cpu]"
+```
+
 ### Supported backends with associated GP packaes
 
 Ready|Backend                                      | GP package
 -----|---------------------------------------------|------------------------------------------
 ✅   |[Tensorflow](https://www.tensorflow.org/)    |[GPflow](https://github.com/GPflow/GPflow)
 ✅   |[PyTorch](https://github.com/pytorch/pytorch)|[GPyTorch](https://gpytorch.ai/)
-✅   |[Numpy](https://numpy.org/)                  |??
-🚧   |[Jax](https://github.com/google/jax)         |??
+✅   |[Numpy](https://numpy.org/)                  | -
+✅   |[JAX](https://github.com/google/jax)         | -
  
+## A basic example
```

Comment:
It may be simpler to just have this script in an examples folder, so that we only provide here the command to run the example. This makes the README shorter and compact.